### ISSUE TITLE
Max backups not working

### ISSUE THREAD

**Issue by PinkingGiovanna** [rthomasv3/Scum-Bag#5]
This might be happening because I'm backing up way too many save files (141)

**Comment by rthomasv3** [comment]
Have you marked any backups as favorites? They don't count toward the max number.

Any more details you can provide would be really helpful - any errors in the logs ("%appdata%\Scum Bag\logs.txt"), what game, how many backups you set as the max, how many you are seeing it create, settings, etc.

**Comment by PinkingGiovanna** [comment]
[logs.txt](https://github.com/rthomasv3/Scum-Bag/files/15210211/logs.txt)
I haven't marked any backups as favorites, I also am saving some save files that aren't from games; I tried to use 3 max backups, 1 max backup but it doesn't work and doesn't delete the old ones.

**Comment by rthomasv3** [comment]
According to the logs, some other program is locking the files and not allowing Scum Bag to access them. This is likely a bug in whatever software you're using alongside Scum Bag, and probably can't be fixed on my side. I'll still look into it further though, because it could be that Scum Bag is the one locking the files elsewhere in the code - so I'll double-check everything.

An explanation of the logs is below.

This log shows an error when checking for changes to a file to see if a backup even needs to happen:
```
[Error]: BackupService>GetHash - System.IO.IOException: The process cannot access the file '.. AppData\Roaming\Achievement Watcher\logs\websocket.log' because it is being used by another process.
```

This logs shows an error when trying to delete old backups and stay within the limit:
```
[Error]: BackupService>CreateBackup - System.IO.IOException: Access to the path '.. AppData\Roaming\Scum Bag\03e8bef5-b789-485c-9634-1dc5d0eebf27\638503676579621143\blob_storage\1b278716-1563-46b8-802d-af7e132c2747' is denied.
   at System.IO.FileSystem.RemoveDirectoryRecursive(String fullPath, WIN32_FIND_DATA& findData, Boolean topLevel)
```

**Comment by PinkingGiovanna** [comment]
Ok, I am using Google Drive to backup the files to it automatically, so it might be because of that. Thanks for the help

**Comment by PinkingGiovanna** [comment]
I was also trying to change the the Backup Directory but it doesn't seem to work. Don't know if it's releated to the same issue as before or if it's something else.

**Comment by rthomasv3** [comment]
Yeah that's probably the same issue. When you change the backup directory it tries to move all the files to the new location, and if google drive locks them, then it would fail with similar errors to the above.

**Comment by PinkingGiovanna** [comment]
So, I disabled Google Drive and the Max backups seem to be working again, but changing the backup directory still doesn't work,  it sometimes moves the content of one directory to the other directory I want but doesn't change the path in the program.

**Comment by PinkingGiovanna** [comment]
I don't know why but even with Drive backup disabled, while playing Elden Ring the max backups still don't work

**Comment by rthomasv3** [comment]
I'm going to take another look at this over the weekend and see what I can do.

**Comment by rthomasv3** [comment]
This should be resolved as of v1.1.5. If not, then there's some other program on your system that's locking the file.
